src/hooks/useFindCities.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react'
+import { fetchData } from '../utils'
+
+export const useFindCities = () => {
```

Comment:
actually i wont add fetch data here unless you add useEffect etc here.
Now you just fetch the data so you dont need to create hook for that